tests/integration/dynamic_ip_test.yaml

```diff
@@ -0,0 +1,1467 @@
+interfaces:
+- name: dhcpcli
+  type: ethernet
+  state: up
+  ipv4:
+    enabled: true
+    dhcp: true
+    auto-dns: true
+    auto-gateway: true
+    auto-routes: true
+    auto-route-table-id: 0
+  ipv6:
+    enabled: true
+    dhcp: true
+    autoconf: true
+    auto-dns: true
+    auto-gateway: true
+    auto-routes: true
+    auto-route-table-id: 0
+description: Configure the dhcpcli ethernet interface with DHCP4, IPv4 auto dns, IPv4 auto gateway, IPv4 auto routes, IPv4 auto route table ID 0, DHCP6, autoconf, IPv6 auto dns, IPv6 auto gateway, IPv6 auto routes, IPv6 auto route table ID 0
+interfaces:
+- name: dhcpcli
+  type: ethernet
+  state: up
+  ipv4:
+    enabled: true
+    dhcp: true
+    auto-dns: true
+    auto-gateway: true
+    auto-routes: true
+    auto-route-table-id: 0
+  ipv6:
+    enabled: true
+    dhcp: true
+    autoconf: true
+    auto-dns: true
+    auto-gateway: true
+    auto-routes: true
+    auto-route-table-id: 0
+description: Remove the interface dhcpcli
+interfaces:
+- name: dhcpcli
+  type: ethernet
+  state: absent
+description: Configure the ethernet device dhcpcli to have IPv4 dhcp-client-id 0f:66:55:BC:73:4D configured
+interfaces:
+- name: dhcpcli
+  type: ethernet
+  state: up
+  ipv4:
+    dhcp-client-id: 0f:66:55:BC:73:4D
+description: Remove the interface dhcpcli
+interfaces:
+- name: dhcpcli
+  type: ethernet
+  state: absent
+description: Configure the dhcpcli ethernet interface with DHCP4, IPv4 auto dns, IPv4 auto gateway, IPv4 auto routes, IPv4 auto route table ID 0, DHCP6, autoconf, IPv6 auto dns, IPv6 auto gateway, IPv6 auto routes, IPv6 auto route table ID 0
+interfaces:
+- name: dhcpcli
+  type: ethernet
+  state: up
+  ipv4:
+    enabled: true
+    dhcp: true
+    auto-dns: true
+    auto-gateway: true
+    auto-routes: true
+    auto-route-table-id: 0
+  ipv6:
+    enabled: true
+    dhcp: true
+    autoconf: true
+    auto-dns: true
+    auto-gateway: true
+    auto-routes: true
+    auto-route-table-id: 0
+description: Configure the ethernet device dhcpcli to have IPv6 dhcp-duid llt configured
+interfaces:
+- name: dhcpcli
+  type: ethernet
+  state: up
+  ipv6:
+    dhcp-duid: llt
+description: Configure the dhcpcli ethernet interface with DHCP4, IPv4 auto dns, IPv4 auto gateway, IPv4 auto routes, IPv4 auto route table ID 0, DHCP6, autoconf, IPv6 auto dns, IPv6 auto gateway, IPv6 auto routes, IPv6 auto route table ID 0
+interfaces:
+- name: dhcpcli
+  type: ethernet
+  state: up
+  ipv4:
+    enabled: true
+    dhcp: true
+    auto-dns: true
+    auto-gateway: true
+    auto-routes: true
+    auto-route-table-id: 0
+  ipv6:
+    enabled: true
+    dhcp: true
+    autoconf: true
+    auto-dns: true
+    auto-gateway: true
+    auto-routes: true
+    auto-route-table-id: 0
+description: Remove the interface dhcpcli
+interfaces:
+- name: dhcpcli
+  type: ethernet
+  state: absent
+description: Configure the ethernet device dhcpcli to have IPv6 dhcp-duid 0f:66:55:BC:73:4D configured
```

Comment:
`Configure ... configured` seems not a good sentence to me. 

`Configure the ethernet device dhcpcli with IPv6 dhcp-duid 0f:66:55:BC:73:4D`